GVFS/GVFS.Installer.Mac/scripts/Distribution.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <installer-gui-script minSpecVersion="1">
+	<title>VFSForGit Mac</title>
```

Comment:
This is a placeholder text, please suggest alternate title. It appears in the Mac installer Wizard, like this -

<img width="627" alt="Screen Shot 2019-03-27 at 12 08 19 PM" src="https://user-images.githubusercontent.com/378580/55092637-46cb9700-5089-11e9-8e50-73ac8f5ca88f.png">